statistics/handle/cache/internal/lfu/lfu_cache.go

```diff
@@ -41,26 +41,33 @@ type LFU struct {
 
 var testMode = false
 
-// NewLFU creates a new LFU cache.
-func NewLFU(totalMemCost int64) (*LFU, error) {
+func setMemCost(totalMemCost int64) (result int64, err error) {
```

Comment:
Better to rename this function to `adjustMemCost` and move `if intest.InTest {...}` in to this function.

Reply:
In fact, when to set the capacity, we don‘t use this ```intest``` logic.